src/lib/browse/layers/CrossingsLayerControl.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { circleRadius } from "colors";
```

Comment:
The dots are hard to hover on, even at tiny zooms. How do we feel about something like `["interpolate", ["linear"], ["zoom"], 1, 2, 8, 3, 13, 10]` from the cycle parking layer? (In one of the other PRs out right now, I refactored another of these zoom-dependent patterns -- we could do the same). Also related to #326 more generally for UX at some zoom levels